src/lib/gql/gql-views.tsx

```diff
@@ -214,6 +214,21 @@ export const getViewPagedItems = async (
       //   totalItems = graphqlResponse.stanfordNews?.pageInfo.total || 0
       //   break
 
+      case "sul_people--randomized_card_grid": {
+        queryVariables.pageSize = 999
+
+        graphqlResponse = await client.stanfordPerson({
+          contextualFilters,
+          ...queryVariables,
+        })
+
+        const pool = graphqlResponse.stanfordPerson?.results as unknown as NodeStanfordPerson[]
+
+        const shuffled = [...pool].sort(() => Math.random() - 0.5)
```

Comment:
This will randomize on every query. You will end up producing duplicates if you query multiple times.